Sources/protoc-gen-swift/MessageGenerator.swift

```diff
@@ -133,8 +122,7 @@ class MessageGenerator {
         }
 
         // Messages that have a storage class will always need @unchecked.
```

Comment:
Comment can go now.

Although it seems like we did "unchecked" when there was storage, and now we don't need that?